services/workflows-service/src/workflow/workflow.service.ts

```diff
@@ -174,7 +177,7 @@ export class WorkflowService {
         },
       }));
     businessId &&
-      (await this.businessRepository.updateById(businessId as string, {
+      (await this.businessRepository.updateById(businessId, {
```

Comment:
It was automatically fixed by ESLint because `businessId` is also inferred as `string`, so the assertion is redundant.